octopose/octopose.py

```diff
@@ -23,23 +23,27 @@
 import argparse
 import sys
 
-from octopose import generate_manifest, deploy
+from octopose import generate_manifest, deploy, __version__
 
 
 def main():
     parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter, prog='octopose',
                                      description='Octopose is a manifest/state driven deployment framework for Octopus Deploy.',
-                                     usage='%(prog)s [generate] [deploy] [-h]',
+                                     usage='%(prog)s [generate] [deploy]',
```

Comment:
would keep the `-h` or `--help` as it shows that is an option ... `git` does this

Reply:
add `--version`